src/components/organisms/Form/FormDropdownFilteredField/FormDropdownFilteredField.test.tsx

```diff
@@ -28,18 +28,20 @@ const nationalities = [{ value: 'British' }, { value: 'Angolan' }];
 const renderComponent = () =>
   render(
     <Form initialValues={{ nationality: '' }} validate={validate} onSubmit={onSubmit}>
-      <Form.DropdownFilteredField
-        name={fieldName}
-        inputProps={{ placeholder: 'Select a nationality...' }}
-        items={nationalities}
-        label={dropdownLabel}
-      />
-      <Form.Button>{buttonLabel}</Form.Button>
+      <Form.Form>
```

Comment:
Form.Form ? Can we not `import { Form, DropdownFilteredField } from '..'` ?

Reply:
I personally also don't like `<Form.Form />` and hopefully we can find a better name in the future. 

Hard to agree that destructuring here is worth though 🤔